README.md

```diff
@@ -16,22 +16,8 @@ Video Demonstrations:
 
 ## Reuse existing cookies and sessions
 
-You can define a specific session ID in order to persist cookies and login information across multiple GPTScript executions.
+You can define a specific workspace directory in order to persist your browser session and cookies across different scripts.
```

Comment:
Maybe give a little more context as to what a workspace directory is? Is it just a directory on disk or something else?